src/App.tsx

```diff
@@ -1,14 +1,104 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
-import { TodoList } from './components/TodoList';
+import { debounce } from 'lodash';
 import { TodoFilter } from './components/TodoFilter';
+import { TodoList } from './components/TodoList';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+export enum FilterBy {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
+
+const getFilteredTodos = (todos: Todo[], filerBy: FilterBy, query: string) => {
+  const filteredTodos = todos.filter(({ title }) => (
+    title.toLowerCase().includes(query.toLowerCase())
+  ));
+
+  if (filerBy !== FilterBy.ALL) {
```

Comment:
you can add this case inside switch / case functionality instead of creating separate if statement